docker-compose.gcp.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  http-api:
+    logging:
+      driver: fluentd
+      options:
+        fluentd-address: 127.0.0.1:24224
```

Comment:
What is fluentd? Is it always exposed on this port?

Reply:
It's a GCP specific thing used for log gather.  That's why it's placed in a separate `docker-compose.gcp.yml` file that's used only on GCP.